pkg/authorization/authorizer/scope/converter.go

```diff
@@ -119,9 +119,12 @@ const (
 	UserInfo        = UserIndicator + "info"
 	UserAccessCheck = UserIndicator + "check-access"
 
+	// UserListProject gives explicit permission to see the projects that this token can see.
+	UserListProject = UserIndicator + "list-projects"
+
 	// UserListProject gives explicit permission to see the projects a user can see.  This is often used to prime secondary ACL systems
 	// unrelated to openshift and to display projects for selection in a secondary UI.
-	UserListProject = UserIndicator + "list-projects"
+	UserListAllProjects = UserIndicator + "list-all-projects"
```

Comment:
> The "list projects my other scopes allow getting" scope is the harder one to understand (and probably less likely to be requested), so I think it has some explaining to do with its name. user:list-scoped-projects? user:list-gettable-projects?

I'm fine with that.  Fun as `list-gettable` would be, I'll probably go with `list-scoped-projects`.